README.md

```diff
@@ -36,9 +36,17 @@ Just create a project, and you’re good to go.
 
 Install it once globally:
 
-```sh
-npm install -g create-react-app
```

Comment:
Hmm, just something I've seen other projects do.

I'm not sure in the value of adding both -- if people have yarn installed, they probably know they want to use it.
Maybe comment the whole thing?

```sh
npm install -g create-react-app
# or: yarn global add create-react-app
```